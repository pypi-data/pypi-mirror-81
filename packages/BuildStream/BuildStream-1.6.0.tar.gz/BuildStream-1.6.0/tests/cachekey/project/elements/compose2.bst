kind: compose
depends:
- filename: elements/import1.bst
  type: build

config:
  integrate: False
