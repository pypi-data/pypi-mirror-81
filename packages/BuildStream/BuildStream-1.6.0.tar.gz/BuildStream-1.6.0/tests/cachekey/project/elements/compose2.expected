7dca9347141d00754c0cc2bbe67695062331002a6a21cd9969435fb70a45600a